src/test/java/com/example/telegrambotinsurance/repositories/UserRepositoryIntegrationTest.java

```diff
@@ -0,0 +1,181 @@
+import org.springframework.beans.factory.annotation.Autowired;
+import com.example.telegrambotinsurance.entities.User;
+import org.springframework.boot.test.context.SpringBootTest;
+
+
+@SpringBootTest
+public class UserRepositoryIntegrationTest {
+	private static final Logger LOGGER = LoggerFactory.getLogger(UserRepositoryIntegrationTest.class);
+	@Autowired
+	private UserRepository userRepository;
+
+	@AfterAll
+	public static void deletingTestObjects(@Autowired UserRepository userRepository) {
+		int deletingCounts = 0;
+		List<User> listOfUsers = userRepository.findAll();
+		for (User user : listOfUsers) {
+			if (user.getEmail().equals("twoBeerOrNotTwoBeer@drunkard.ru")) {
+				userRepository.delete(user);
+				deletingCounts++;
+			}
+		}
+		LOGGER.debug("Count of deleting operations  after all tests: " + deletingCounts);
+	}
+
+	@AfterEach
+	public void deletingTestObjectsAfterMethod() {
+		int deletingCounts = 0;
+		List<User> listOfUsers = userRepository.findAll();
+		for (User user : listOfUsers) {
+			if (user.getEmail().equals("twoBeerOrNotTwoBeer@drunkard.ru")) {
+				userRepository.delete(user);
+				deletingCounts++;
+			}
+		}
+		LOGGER.debug("Count of deleting operations: " + deletingCounts);
+	}
+
+	@Test
+	public void whenCalledSave_thenNewUserSavedCorrectly() {
+		String name = String.valueOf(System.currentTimeMillis());
+		String email = "twoBeerOrNotTwoBeer@drunkard.ru";
+		User user = new User(name, email);
+		long listSizeBeforeSaving;
+		long listSizeAfterSaving;
+
+		//List users before save
+		List<User> listOfUsersBeforeSavingNewUser = userRepository.findAll();
+
+		//user`s size field`s
+		listSizeBeforeSaving = listOfUsersBeforeSavingNewUser.size();
+		for (User userList : listOfUsersBeforeSavingNewUser
+		) {
+			LOGGER.debug("BEFORE SAVING: " + userList);
+		}
+		LOGGER.debug("**********");
+
+		//saving new user
+		Assertions.assertFalse(listOfUsersBeforeSavingNewUser.contains(user));
+		userRepository.save(user);
+
+		//user`s after save
+		List<User> listOfUsersAfterSavingNewUser = userRepository.findAll();
+		listSizeAfterSaving = listOfUsersAfterSavingNewUser.size();
+		for (User userList : listOfUsersAfterSavingNewUser
+		) {
+			LOGGER.debug("AFTER SAVING: " + userList);
+		}
+
+		//assertion
+		Assertions.assertTrue(listOfUsersAfterSavingNewUser.contains(user));
+		Assertions.assertEquals(listSizeBeforeSaving + 1, listSizeAfterSaving);
+	}
+
+	@Test
+	public void whenCalledDelete_thenDeletingUserCorrectly() {
+		String name = String.valueOf(System.currentTimeMillis());
+		String email = "twoBeerOrNotTwoBeer@drunkard.ru";
+		User user = new User(name, email);
+		long listSizeBeforeSaving;
+		long listSizeBeforeDeleting;
+		long listSizeAfterDeleting;
+
+		// taking data list
+		List<User> listOfUsersBeforeSavingUser = userRepository.findAll();
+		Assertions.assertFalse(listOfUsersBeforeSavingUser.contains(user));
+		listSizeBeforeSaving = listOfUsersBeforeSavingUser.size();
+
+		//saving new user
```

Comment:
не очень информативный коммент. по названию метода это и так понятно :)

лучше бы написал зачем ты тут сохраняешь юзера
```suggestion
		// saving new user to test update on it
```

Reply:
и это?

Reply:
второй раз отправляешь на ревью этот код без изменений... если не согласен с чем-то - ну так напиши, а не просто игнорируй

Reply:
Прошу прощения, исправил